components/use-toasts/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`UseToast should render different actions 1`] = `
-"<div class=\\"toast-container \\"><div class=\\"toast  \\"><div class=\\"message\\">hello</div><div class=\\"action\\"><button type=\\"button\\" class=\\"btn \\"><div class=\\"text\\">remove</div><style>
+"<div class=\\"toast-container \\"><div class=\\"toast visible \\"><div class=\\"message\\">hello</div><div class=\\"action\\"><button type=\\"button\\" class=\\"btn \\"><div class=\\"text\\">remove</div><style>
```

Comment:
What's wrong with the snapshot here?

Reply:
i didn't touch that, i just use `jest --update` from `package.json` after i made my test, so maybe some tests were not up to date?

Reply:
Use `yarn test-update` instead of `jest --update`.

Reply:
that's the one i used :)

Reply:
Try to update the code or rebase branch? I tried again with the latest code and didn't change the file.
(If you really can't solve it, you can choose to ignore this file and submit the code)

Reply:
i rebase on master, i think we are good for this one now.